### PR TITLE
chore(ci): add firefox support for wasm tests and benchmarks

### PR DESCRIPTION
**Pull request by soonum**

### CODE REVIEW

File: .github/workflows/benchmark_wasm_client.yml [hunk 1]
```diff
@@ -106,12 +106,12 @@ jobs:
...
           make install_node
-          make install_chrome_browser
-          make install_chrome_web_driver
+          make install_firefox_browser
+          make install_firefox_web_driver
 
       - name: Run benchmarks
         run: |
-          make bench_web_js_api_parallel_chrome_ci
+          make bench_web_js_api_parallel_firefox_ci
```
Comment (IceTDrinker) [inline .github/workflows/benchmark_wasm_client.yml:114]: could we have both benchmarks ? at least initially to be able to compare ?

**Review by IceTDrinker — COMMENTED** [review]
just a small thing

File: .github/workflows/benchmark_wasm_client.yml (hunk 1 above)
Comment (soonum) [reply .github/workflows/benchmark_wasm_client.yml]: I've put a mechanism to do so.

**Review by IceTDrinker — APPROVED** [review]
Alright let's try this, do you have a backlog issue for updating Selenium and reducing the stdout spam ?

**Pull request merged**